catalogs/usfimr-s1/README.md

```diff
@@ -0,0 +1,43 @@
+# USFIMR Correlated S1 Imagery STAC Catalog
+
+This project is responsible for generating an S1 dataset and the corresponding STAC catalog. This S1
+imagery is correlated with any matching floods from the [USFIMR dataset](https://cfim.ornl.gov/data/).
+
+## Data ingest
+
+A Sentinel Hub account with beta access to batch requests is necessary to complete the ingest. Once
+that is available, an oauth client can be generated from within the user dashboard. This will provide
+a user ID and a secret which jointly are sufficient to generate the token we will need to carry out
+Sentinel Hub API calls.
+
+Prepare the EU bucket following [these instructions](https://docs.sentinel-hub.com/api/latest/api/batch/#aws-s3-bucket-settings)
```

Comment:
Could note here that a bucket is available in the noaa-flood AWS account already at

`s3://noaafloodmapping-sentinelhub-batch-eu-central-1`